core/storage/collection/gae_models.py

```diff
@@ -323,7 +327,7 @@ def compute_models_to_commit(
             additional_models
         )
 
-        # The cast is needed because the additional_models is list of BaseModels
+        # Here we use cast because the additional_models is list of BaseModels
         # and we want to hint the mypy that this is CollectionRightsModel.
```

Comment:
the --> to

Also let's be consistent everywhere: MyPy instead of mypy. (Please fix elsewhere if needed.)

Reply:
Done everywhere it is applicable !!